docs/changelog/16_0_0.md

```diff
@@ -17,3 +16,25 @@ Here are some of our feature highlights:
 {% card title="Task Graph Improvements"  type="video" url="https://youtu.be/9_Y6Mop-Kac" /%}
 {% card title="IntelliJ Nx Console Feature Parity"  type="video" url="https://youtu.be/XCoeNiyM6hw" /%}
 {% /cards %}
+
+## Deprecations
+
+{% cards cols="2" %}
+{% card title="workspace-lint" type="external" url="https://github.com/nrwl/nx/pull/16385" /%}
+{% card title="module federation utils from devkit public api" type="external" url="https://github.com/nrwl/nx/pull/16574" /%}
+{% card title="Global implicitDependencies" url="/deprecated/global-implicit-dependencies" /%}
+{% card title="stylus style option for react and next" type="external" url="https://github.com/nrwl/nx/pull/16135" /%}
+{% card title="@nrwl/tao package" type="external" url="https://github.com/nrwl/nx/pull/16550" /%}
+{% /cards %}
+
+## Breaking Changes
```

Comment:
We could even think about having a reusable component for these that renders them always the same way, as we're going to have again and again. But we can do that at a later point too.

The only thing I'm thinking here is whether we should have here an intro sentence/note that mentions to use the `nx migrate` (with link to migrate docs) to automatically be upgraded avoiding these breaking changes.